barretenberg/ts/src/barretenberg_api/index.ts

```diff
@@ -1140,6 +1200,30 @@ export class BarretenbergApiSync {
     return out[0];
   }
 
+  acirVerifyUltraStarknetHonk(proofBuf: Uint8Array, vkBuf: Uint8Array): boolean {
```

Comment:
This function was missing @feltroidprime though the async version of it was previously present